packages/nc-gui/components/smartsheet/toolbar/ColumnFilter.vue

```diff
@@ -219,7 +249,7 @@ defineExpose({
               dropdown-class-name="nc-dropdown-filter-comp-op"
               @change="filterUpdateCondition(filter, i)"
             >
-              <template v-for="compOp of comparisonOpList" :key="compOp.value">
+              <template v-for="compOp of comparisonOpList(getColumn(filter)?.uidt)" :key="compOp.value">
```

Comment:
the `key` should be unique across the loop so use index or id since the value could be identical.

Reply:
> since the value could be identical.

Any example? I don't see the list contains duplicate values.

Reply:
@wingkwong - sorry I thought it was filter look... here it won't cause an issue :+1: